gama/genetic_programming/mutation.py

```diff
@@ -151,8 +230,16 @@ def random_valid_mutation_in_place(
         )
     else:
         replaceable_primitives = filter(
-            lambda p: len(primitive_set[p._primitive.output]) > 1, individual.primitives
+            lambda p: has_multiple_options(
+                config_space.get_hyperparameter(
+                    config_space.meta[p._primitive.output]
+                    if p._primitive.output in get_internal_output_types()
+                    else p._primitive.output
```

Comment:
I'm missing part of the picture here, can you give me an example of output for the `p._primitive.output` that is not one of the `get_internal_output_types()`?